po/de.po

```diff
@@ -20026,16 +20022,16 @@ msgid "unable to write tag file"
 msgstr "konnte Tag-Datei nicht schreiben"
 
 #: builtin/tag.c:210
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "You have created a nested tag. The object referred to by your new tag is\n"
 "already a tag. If you meant to tag the object that it points to, use:\n"
 "\n"
 "\tgit tag -f %s %s^{}"
 msgstr ""
-"Sie haben einen verschachtelten Tag erzeugt. Das Objekt, welches Sie\n"
-"angegeben haben, ist bereits ein Tag. Wenn Sie das Objekt taggen wollten,\n"
-"worauf dieses zeigt, nutzen Sie:\n"
+"Sie haben einen verschachtelten Tag erzeugt. Das Objekt, welches Sie über\n"
+"den neuen Tag angegeben haben, ist bereits ein Tag. Wenn Sie das Objekt\n"
```

Comment:
"Das Objekt, auf welches Ihr neues Tag referenziert, ist bereits ein Tag." ?

Reply:
I've taken your translation but changed it to "Ihr neuer Tag referenziert"...

Reply:
Tag wird hier in der Übersetzung als "das Tag" bezeichnet, im Gegensatz zu "der Tag".